server/src/data/data.service.ts

```diff
@@ -245,23 +245,113 @@ export class DataService {
     });
   }
 
+  async columUpdateStream(type: string, fieldName?: string, bodyData?: Column) {
+    const columnFileData: Array<Column> = [];
+    let isDataTypeChanged: boolean = false;
+    await new Promise((resolve, reject) => {
+      const importStream = readFileStream(),
+        columnDataParser = JSONStream.parse('columns.*');
+      importStream.pipe(columnDataParser);
+      columnDataParser.on('error', (error) => {
+        reject(error);
+      });
+      columnDataParser.on('data', (columnData: Column) => {
+        if (type === File.UPDATE_COLUMN) {
+          if (
+            columnData.fieldName === fieldName &&
+            columnData.dataType !== bodyData.dataType
+          ) {
+            isDataTypeChanged = true;
```

Comment:
Change this to:
isDataTypeChanged = (columnData.fieldName === fieldName &&
            columnData.dataType !== bodyData.dataType)